Database/database.js

```diff
@@ -24,4 +25,6 @@ const reviewSchema = new mongoose.Schema({
   location: String
 });
 
-module.exports= mongoose.model('Review', reviewSchema);
+reviewSchema.index({bookId: 1, reviewerId: 1}, {unique: true});
```

Comment:
Interesting, what is this index for?

Reply:
To ensure that there is only one author on each review for each book